Dockerfile

```diff
@@ -8,11 +8,29 @@ RUN apt update && \
 COPY files/usr/local/bin/entrypoint /usr/local/bin/entrypoint
 
 # Install peertube-runner
-RUN npm install -g @peertube/peertube-runner@${PEERTUBE_RUNNER_VERSION:-"0.0.9"}
+RUN npm install -g @peertube/peertube-runner@${PEERTUBE_RUNNER_VERSION:-"0.0.17"}
 
 # Un-privileged user running the application
 ARG DOCKER_USER
 USER ${DOCKER_USER}
 
 ENTRYPOINT [ "entrypoint" ]
-CMD [ "peertube-runner", "server" ]
\ No newline at end of file
+CMD [ "peertube-runner", "server" ]
+
+FROM runner AS whisper_ctranslate2
+
+USER root:root
+
+RUN apt-get update && \
+    apt install -y python3-pip && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt install python3-pip -y
+
+RUN pip3 install whisper-ctranslate2==0.4.5 --break-system-packages
```

Comment:
I don't like this way to install it. It is also possible to set the binary path to the runner. IMO you should install it a proper way and set the binary path in the configuration file created in the entrypoint.

Reply:
I don't get it… What is a proper way ?

Reply:
What did you mean by ?
> you should install it a proper